### PR TITLE
Fix bug 1398108: Do not duplicate group type in group name

### PR DESCRIPTION
**Pull request by mathjazz**
@jotes r?

### CODE REVIEW

File: pontoon/base/migrations/0103_bug_1398108_rename_groups_with_duplicate_words.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+
+def rename_groups_with_duplicate_words(apps, schema_editor):
+    Group = apps.get_model('auth', 'Group')
+    groups_to_update = []
+
+    for name in ('managers', 'translators'):
+        double = '{} {}'.format(name, name)
+        for group in Group.objects.filter(name__contains=double):
+            group.name = group.name.replace(double, name)
```
Comment (jotes) [inline pontoon/base/migrations/0103_bug_1398108_rename_groups_with_duplicate_words.py]: I think you could try to use  `.update()` here.

Comment (mathjazz) [reply pontoon/base/migrations/0103_bug_1398108_rename_groups_with_duplicate_words.py]: I tried that in combination with the `F()` expression, but failed to find a solution for search and replace.

Comment (jotes) [reply pontoon/base/migrations/0103_bug_1398108_rename_groups_with_duplicate_words.py]: https://gist.github.com/simonw/30d2e626772e84edd688ba717a766c89

Comment (mathjazz) [reply pontoon/base/migrations/0103_bug_1398108_rename_groups_with_duplicate_words.py]: Thanks! Updated.

File: pontoon/base/signals.py [hunk 2]
```diff
@@ -94,8 +94,8 @@ def create_locale_permissions_groups(sender, **kwargs):
         return
 
     try:
-        create_group(instance, 'translators', ['can_translate_locale'], '{} translators'.format(instance.code))
-        create_group(instance, 'managers', ['can_translate_locale', 'can_manage_locale'], '{} managers'.format(instance.code))  # noqa
+        create_group(instance, 'translators', ['can_translate_locale'], instance.code)
+        create_group(instance, 'managers', ['can_translate_locale', 'can_manage_locale'], instance.code)  # noqa
```
Comment (adngdb) [inline pontoon/base/signals.py]: What this ``# noqa`` for? I guess you can drop it now.

Comment (mathjazz) [reply pontoon/base/signals.py]: The line is too long. I'll fix that.

**Pull request merged**